src/modules/addDeleteTask.js

```diff
@@ -0,0 +1,68 @@
+import threeDots from './three-dots.js';
+import editTask from './edit-task.js';
+import tasks from './task-obj.js';
+
+// Declare function to add & delete tasks in the list
+const addTask = (task) => {
+  const list = document.querySelector('.todo_list');
+
+  const temp = ` <li class="list_item common_styles">
+        <div class="task_container">
+            <input type="checkbox" id="${task.id}">
+            <label>${task.description}</label>
+        </div>   
+           <div class="dot_div" id="${task.id}">
+           <ul class="dot_ul">
+               <li class="dot_li">
+                     <button class="dot_btn edit_btn" id="">
+                     <img src="./images/edit.png" alt="edit" class="edit_icon"></img>
+                      Edit ask
+                     </button>
+               </li>
+                <hr>
+               <li class="dot_li">
+                     <button class="dot_btn trash_btn" id="${task.index}">
+                     <img src="./images/trash.png" alt="trash" class="trash_icon"></img>
+                     Delete task
+                     </button>
+               </li>
+           </ul>
+         </div>
+         <img src="./images/dots.png" alt="dots" class="dots_icon" id="${task.id}"</img>  
+    </li>`;
+
+  list.innerHTML += temp;
+
+  threeDots();
+
+  editTask();
```

Comment:
- [ ] Kindly fix the `edit` task functionality by updating the local storage when you edit any task as now when I press the edit button then enter a new value for a task and try to refresh the page the task remains the same as the old state.